### PR TITLE
Feature - respond with notifications

### PR DESCRIPTION
**Pull request by joefitter**
* Added notifications endpoint to respond with notifications from db
* Added asl-schema
* Updated drone to 1.0

### CODE REVIEW

File: .drone-1.0.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  - name: docker build
+    image: docker:dind
+    environment:
+      DOCKER_HOST: tcp://docker:2375
+      NPM_AUTH_USERNAME:
+        from_secret: npm_auth_username
+      NPM_AUTH_TOKEN:
+        from_secret: npm_auth_token
+    commands:
+      - docker build --build-arg NPM_AUTH_USERNAME=$${NPM_AUTH_USERNAME} --build-arg NPM_AUTH_TOKEN=$${NPM_AUTH_TOKEN} -t asl-stub .
```
Comment (lennym) [inline .drone-1.0.yml:42]:
```suggestion
      # wait for docker service to be up before running docker build
      - n=0; while [ "$n" -lt 60 ] && [[ ! $(docker stats --no-stream) ]]; do n=$(( n + 1 )); sleep 1; done
      - docker build --build-arg NPM_AUTH_USERNAME=$${NPM_AUTH_USERNAME} --build-arg NPM_AUTH_TOKEN=$${NPM_AUTH_TOKEN} -t asl-stub .
```

File: .drone-1.0.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+services:
+- name: docker
+  image: docker:dind
+  environment:
+    DOCKER_TLS_CERTDIR: ""
+- name: postgres
+  image: postgres
+  environment:
+    POSTGRES_USER: asl-test
+    POSTGRES_PASSWORD: test-password
```
Comment (lennym) [inline .drone-1.0.yml:70]: The postgres service isn't being used by any build steps, so should be removed.

Comment (joefitter) [reply .drone-1.0.yml:72]: 👍

File: .drone-1.0.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      NPM_AUTH_USERNAME:
+        from_secret: npm_auth_username
+      NPM_AUTH_TOKEN:
+        from_secret: npm_auth_token
+      DOCKER_PASSWORD:
+        from_secret: docker_password
+    commands:
+      - docker login -u="ukhomeofficedigital+asl" -p=$${DOCKER_PASSWORD} quay.io
+      - docker tag asl-stub quay.io/ukhomeofficedigital/asl-stub:$${DRONE_COMMIT_SHA}
+      - docker push quay.io/ukhomeofficedigital/asl-stub:$${DRONE_COMMIT_SHA}
```
Comment (lennym) [inline .drone-1.0.yml:58]: This was pushing `latest` before. It's _probably_ better to push the sha, but we'll need to change some of the downstream config to load the version.

Comment (joefitter) [reply .drone-1.0.yml:58]: 👍

**Pull request merged**